webpack.config.js

```diff
@@ -30,10 +30,11 @@ var config = {
           presets: ['react', 'es2015'],
           plugins: [
             'transform-class-properties',
-            'transform-object-rest-spread'
+            'transform-object-rest-spread',
+            'transform-exponentiation-operator'
           ]
         },
-        include: path.join(__dirname, 'src')
+        include: [path.join(__dirname, 'src'), path.resolve(__dirname, "node_modules/recharts/es6")] //recharts es6 needed
```

Comment:
why is it necessary to add this resolve path?

Reply:
It was meant as a temporary workaround for some issues in recharts sources that have since been fixed. I will remove it.